README.md

```diff
@@ -2,8 +2,75 @@
+## Vision
+Open source software is amazing, and so are the creators and contributors who share it.
+How amazing? It's difficult to tell, since we don't have good
+tools for recognizing those people. Many amazing open-source
+contributors labor in the shadows, going unappreciate for the work they do.
 
-SourceCred will create a "Cred Graph", which is a graph that shows how the contributions that compose open-source projects are related to and derive value from each other. From this, we'll be able to assign "cred" to users based on how valuable their contributions are. Cred will be assigned based on a mixture of objective data (e.g. references between GitHub pull requests) and subjective feedback (e.g. projects' own judgments on how important different contributions were).
+As the open economy develops, we need to go beyond [commit streaks] and
+follower counts. We need transparent, accurate, and fair tools for recognizing
+and rewarding open collaboration. SourceCred aims to do that.
 
-If you'd like to contribute, please follow along with [our issues](https://github.com/sourcecred/sourcecred/issues), as we are using issues to coordinate development and design decisions. We also have [a slack](https://join.slack.com/t/sourcecred/shared_invite/enQtMzA4NzI5ODIwODMyLWFiNDlhNWNiODc4MTk4MjNmZTAzMDNjNDAwYzEyZTBiNjAxZTFhMjU1MDg2YzNlN2FlNzgwYmU0NGM1NGEzM2M).
+[commit streaks]: https://www.mxsasha.eu/blog/2016/04/01/how-github-contribution-graph-is-harmful/
+
+SourceCred will enable projects to create and track "cred", which is a
+quantitative measure of how much value different contributors added to a
+project. We'll do this by providing a basic data structure—a [cred graph] -
+into which projects can add all kinds of information about the contributions
+that compose it. For example, a software project might include information
+about GitHub pull requests, function declarations and implementations, design
+documents, community support, documentation, and so forth.  We'll also provide
+an algorithm ([PageRank]) which will ingest all of this information and produce
+a "cred attribution", which assigns a cred value to each contribution, and thus
+to the people who authored the contributions.
+
+[cred graph]: https://en.wikipedia.org/wiki/Directed_graph
+[PageRank]: https://en.wikipedia.org/wiki/PageRank
+
+## Principles
+
+1. **Transparent**
+
+   If it's to be a legitimate and accepted way of tracking credit in projects,
+   cred attribution can't be a black-box. SourceCred will provide tools that
+   make it easy to dive into the cred attribution, and see exactly why
+   contributions were valued the way they were.
+
+2. **Community-controlled**
+
+   At the end of the day, the community of collaborators in a project will know
+   best which contributions were important and deserve the most cred. To think
+   an algorithm would discover that on its own is hubris. To that end, we'll
```

Comment:
Do you want to add a qualifier, like “…discover that perfectly on its own…”? The goal _is_ to discover that automatically as best as possible; we just accept that it will be imperfect and welcome manual adjustments.

Reply:
Done